### PR TITLE
document how to deal with mix dependencies

### PR DESCRIPTION
**Pull request by the-mikedavis**

### CODE REVIEW

**Comment by ttanveer-nfi** [comment]
@the-mikedavis words are hard, see what you think.

File: guides/Mix Dependencies.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+Instead of adding the events and relay_commands dependencies to the `mix.exs` 
+file like we would normally do, we add it to the list of `explicit_dependencies` 
+in the `harness.exs` file.
+
+Example:
+```
+config :harness_aggregate,
+  app: :planning_schedule,
+  aggregate_type: "planning.schedule",
```
Comment (the-mikedavis) [inline guides/Mix Dependencies.md:16]:
```suggestion
  aggregate_type: "Planning.Schedule",
```

you probably wanna change this in the Planning.Schedule repo as well

should be fixed in the gaas generator to enforce this formatting but /shrug that takes work. I'll get around to it eventually

**Review by the-mikedavis — COMMENTED** [review]
looks good

thx for doin this

apparently I can't approve my own PR so here's the :white_check_mark: :

:heavy_check_mark:

**Comment by the-mikedavis** [comment]
I think I might graft these docs onto a harness package like NFIBrokerage/harness_aggregate instead of keeping it in harness itself

it's a tough call because it's not just harness_aggregate that manages the `mix.exs` project and dependencies (e.g. harness_micro_controller does the same)

but this repo is published publicly and these docs pertain to our private dependencies on our private hex

I think if this info is in the harness_aggregate readme, we can just point to it in harness_process_manager et. al. and that'll be good enough

**Comment by the-mikedavis** [comment]
viola!: https://github.com/NFIBrokerage/harness_aggregate#dependencies

**Pull request closed without merging**